geps/gep-1742.md

```diff
@@ -419,14 +426,22 @@ type HTTPRouteTimeouts struct {
 	// may result in more than one call from the gateway to the destination backend service,
 	// for example, if automatic retries are supported.
 	//
-	// Because the Request timeout encompasses the BackendRequest timeout,
-	// the value of BackendRequest defaults to and must be <= the value of Request timeout.
+	// This value follows a subset of Golang and XML Duration formatting, for example 1m, 1s,
```

Comment:
This applies to text above this line.  Can you add an explanation about whether the Request time encompasses/includes the BackendRequest time? Is Request = BackendRequest + other processing time, or is total time = Request + BackendRequest, implying that Request does NOT include the BackendRequest time?

Reply:
Another thought. I understand that we can't enforce that `BackendRequest timeout must be <= Request timeout`, but maybe we could/should still say something like this?
```suggestion
	// Because the Request timeout encompasses the BackendRequest timeout,
	// the value of BackendRequest should be <= the value of Request timeout to have any effect.
	// This value follows a subset of Golang Duration formatting, for example 1m, 1s,
```